configuration/data/definitions.ts

```diff
@@ -0,0 +1,138 @@
+export type NomadIdentifier = string;
```

Comment:
Need to update definitions. Leaving for later PR due to avoid adding more scope to already overloaded PR: https://github.com/nomad-xyz/rust/issues/39